### PR TITLE
pay it forward from 5.0.2 PR #3576

### PR DESCRIPTION
**Pull request by sjpadgett**
- Replace nbsp with unknown
- fixup encounter where non-auth creator is reflected in provider list.
- add BS alert to Lists for dup lists
- fix broken delete in List

### CODE REVIEW

File: interface/super/edit_list.php [hunk 1]
```diff
@@ -1036,7 +1036,7 @@ function mysubmit() {
             <div class="collapse navbar-collapse" id="navbar-list">
                 <ul class="nav navbar-nav">
                     <li class="nav-item"><a class="nav-link" href="#" data-toggle="modal" data-target="#modal-new-list"><i class="fa fa-plus"></i>&nbsp;<?php echo xlt('New List'); ?></a></li>
-                    <li class="nav-item"><a class="nav-link" href="#" class="deletelist" id="<?php echo attr($list_id); ?>"><i class="fa fa-trash"></i>&nbsp;<?php echo xlt('Delete List'); ?></a></li>
+                    <li class="nav-item"><a class="nav-link deletelist" href="#" id="<?php echo attr($list_id); ?>"><i class="fa fa-trash"></i>&nbsp;<?php echo xlt('Delete List'); ?></a></li>
```
Comment (sjpadgett) [inline interface/super/edit_list.php:1039]: @tywrenn Any way to add a check for this error case? I've fixed several of these and they tend to be in our li updates.

**Comment by sjpadgett** [comment]
okay tired of fixing issues. moving on.......

File: interface/super/edit_list.php (hunk 1 above)
Comment (tywrenn) [reply interface/super/edit_list.php:1039]: Haven't found a linter yet that finds this -- this is a very easy mistake to have.

Comment (tywrenn) [reply interface/super/edit_list.php:1039]: Haven't found a linter yet that finds this -- this is a very easy mistake to have.

**Pull request merged**